src/mail.sh

```diff
@@ -450,6 +553,7 @@ function parse_mail_options()
   options_values['FORCE']=0
   options_values['VERIFY']=0
   options_values['TEMPLATE']=''
+  options_values['INTERACTIVE']=0
```

Comment:
Can we set `''` instead of `0` and use `-n` or `-z` to check this value in the first part of this commit?

Reply:
We can, I used `0` and `1` because this is just a flag, but I think I'll change this a bit and maybe it'll make more sense as an empty string